src/module/Provider.sol

```diff
@@ -23,7 +23,7 @@ contract Provider is IProvider {
         } else if (asset == MC.BUFFER) {
             return IERC4626(MC.BUFFER).previewRedeem(1e18);
         } else if (asset == MC.STETH) {
-            return IStETH(MC.STETH).getPooledEthByShares(1e18);
+            return 1e18;
```

Comment:
for STETH the ratio here is 1:1

for WSTETH we need another entry

Reply:
ok good